webapp/src/pages/EditUserPage/EditUserPage.css

```diff
@@ -0,0 +1,58 @@
+.edit-user-page {
+  margin: 0;
+  padding: var(--padding-top-bottom) var(--padding-sides);
+  height: 100vh;
+  background-color: var(--color-background);
+  color: var(--color-light);
+  display: grid;
+  gap: 4rem var(--column-gap);
+  grid-template-columns: 1fr;
+  grid-template-rows: auto auto 1fr auto;
+  overflow: scroll;
+}
+
+.edit-user-form {
+  min-width: 15rem;
+  max-width: 20rem;
+  justify-self: center;
+}
+
+.edit-user-form-button {
+  justify-self: center;
+  align-self: end;
+}
+
+@media all and (min-width: 768px) {
+  .edit-user-page {
+    display: grid;
```

Comment:
if you are not overriding the options of the mobile version, it's useless to set it inside the media query. I.e.: we already set `edit-user-page` to `display: grid` in line 7. We don't need to do it again.

Reply:
Yes, you're right! Thnaks